tools/depends/target/Makefile

```diff
@@ -147,6 +158,7 @@ test-dependencies:
 
 
 distclean::
+	rm $(PREFIX)/share/cross-file.meson
```

Comment:
should this be in a separate commit? It seems unrelated to this PR.

Reply:
Yeah, there's a couple things that are in here in the same boat. Been a long wild ride getting to this point with a lot of changes, haha.

Reply:
Done, separated out.